tests/serialization/identified_test.py

```diff
@@ -63,7 +63,7 @@ def read_data(self, inp):
         self.a_float = inp.read_float()
         self.a_double = inp.read_double()
 
-        self.bytes_ = inp.read_byte_array()
```

Comment:
minor comment: these test changes should not be required anymore and maybe you can add an extra field to test the behaviour when the return value is bytearray